Projects/Feature/MainFeature/Sources/Scene/MainView.swift

```diff
@@ -144,8 +144,8 @@ struct MainView: View {
                 }
             }
             .smsAlert(
-                title: "게스트 종료",
-                description: "게스트 이용을 종료하시겠습니까?",
+                title: "로그아웃",
+                description: "정말로 로그아웃 하시겠습니까?",
```

Comment:
```suggestion
                description: "정말로 로그아웃하시겠습니까?",
```
이 예의 '-하다'는 앞의 명사와 붙여야 합니다. 그러므로 로그아웃하시겠습니까?로 붙여주세요.